homework/style.css

```diff
@@ -1,3 +1,90 @@
 .alert-error {
     color: red;
-}
\ No newline at end of file
+    ba
+}
+
+body {
+  background-color: rgba(142, 93, 93, 1);
+}
+#root {
+  display: flex;
+   flex-direction: column;
+  align-items: flex-start;
+    justify-content: center;
+}
+.header {
+  background-color: #444;
+  width: 100%;
+}
+.h1 {
+  text-align: center;
+
+}
+.container {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  margin: 1 rem;
```

Comment:
I think that this doesn't do anything since there is a space between `1` and `rem`.